FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/common/main.c

```diff
@@ -89,7 +89,7 @@ static const uint8_t ucDNSServerAddress[ 4 ] = { configDNS_SERVER_ADDR0, configD
  * name of the constant.  Options include to standard out (xLogToStdout), to a disk
  * file (xLogToFile), and to a UDP port (xLogToUDP).  If xLogToUDP is set to pdTRUE
  * then UDP messages are sent to the IP address configured as the echo server
```

Comment:
configured as the **UDP logging server**

Reply:
Fixed